### ISSUE TITLE
[Radio buttons] radio buttons do not show visible focus styles

### ISSUE THREAD

**Issue by carodew** [18F/usfwds#413]
**Steps to recreate:**

Set focus on one of the radio buttons. No focus style is visible. 

**Expected behavior:**
I would expect radio buttons (selected or not) to show a visible focus style. 

**Tested in**
Macbook Pro
Yosemite 10.10.5
Firefox 39.0.3
staging

**Comment by carodew** [comment]
Actually I'm having trouble tracking this down in the real world, seems like a lot of radio buttons don't have visible focus styles. This might not be a bug, but that seems wrong to me. Currently awaiting further clarification from the accessibility guild.

**Comment by carodew** [comment]
@nickbristow says "always focus!"

**Comment by carodew** [comment]
@mollieru and @angel – can you provide a visual design for the focus state?

**Comment by angel** [comment]
@carodew Can we do the same blue focus we are doing for the text input?

**Comment by carodew** [comment]
I think that's ultimately your decision.

**Comment by maya** [comment]
@carodew it does have a focus. This is it:

![screen shot 2015-09-03 at 2 58 39 pm](https://cloud.githubusercontent.com/assets/5249443/9672067/4e39fd72-524c-11e5-87a4-8933109de6b3.png)

**Comment by carodew** [comment]
Ok. well, I can't tab through them on the keyboard or if I am tabbing, there is no visual indication of where focus is on radio buttons.

**Comment by maya** [comment]
It's up and down keys for radio buttons

**Comment by carodew** [comment]
I guess the issue is that if I set focus on a checkbox above it and I see a distinct focus state I know where focus is on the page. If I tab through I expect the focus to move to the radio button group in some way, but there is no focus state (even on the selected radio button) to tell me where focus is on the page. So it feels like I'm lost. @nickbristow – any thoughts?

**Comment by carodew** [comment]
So basically, all we need is a version of the selected radio button that also has focus. A deselected radio button would never have a focus style.

**Comment by maya** [comment]
@carodew fixed in https://github.com/18F/usfwds/pull/479. Can you please test and confirm?

**Comment by carodew** [comment]
@jpyuda tested and confirmed. Closing this issue!
